dev/scripts/create-mysql-container.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+
+docker run -d --name dev-surface-mysql \
+              -p3306:3306 \
+              -e MYSQL_INNODB_BUFFER_POOL_SIZE=512M \
+              -e MYSQL_DATABASE=surface \
+              docker.app.betfair/appsec/surface/mysql-dev:8.0.30-18
```

Comment:
here is an internal reference, can be a public alternative?

Reply:
Indeed. Thanks for the review. Updated.